ci-operator/jobs/infra-periodics.yaml

```diff
@@ -717,15 +717,15 @@ periodics:
           secretKeyRef:
             key: url
             name: ci-slack-api-url
-      image: registry.svc.ci.openshift.org/ci/applyconfig:latest
+      image: applyconfig:latest
```

Comment:
What makes `applyconfig` avaialble on app.ci? Even the presubmits and postsubmits are using full pullspec.

Reply:
The imagestream exists in the `ci` namespace and its `local: true` as well

Reply:
That's not what I asked... I'm trying to understand on what are we relying on here. The IS does not seem to have a manifest in the repo, what is making sure it exists?

Reply:
This is from the test-image distributor, which is configured to distribute everything in the `ci` namespace: https://github.com/openshift/release/tree/b2770007d21ea33f7ac1ea2ad780904e127448b9/clusters/app.ci/prow